satisfactory-accounting-app/src/node_display/group/group_name.rs

```diff
@@ -68,6 +76,9 @@ impl Component for GroupName {
                     false
                 }
             }
+            Msg::NoOp => {
+                true
```

Comment:
You shouldn't need the NoOp (see below about using `batch_callback`), but for future reference, the boolean here is whether or not the component needs to be re-rendered as a result of the update.

Reply:
Ah - that's helpful. Thanks. 👍